### PR TITLE
Add in-memory file handler

### PR DESCRIPTION
**Pull request by ghost**
Map file has been rebuilt.

### CODE REVIEW

File: src/inmemoryfile.cpp [hunk 1]
```diff
@@ -0,0 +1,63 @@
+#include "inmemoryfile.hpp"
```
Comment (dawidrylko) [inline src/inmemoryfile.cpp:1]: files names convention? 🤔

Comment (akondas) [reply src/inmemoryfile.cpp:1]: what do you mean by that, my friend?

**Review by akondas — APPROVED** [review]
Perfect :medal_military:

File: src/inmemoryfile.cpp (hunk 1 above)
Comment (dawidrylko) [reply src/inmemoryfile.cpp:1]: other files names -> snake_case
Zrzut ekranu 2019-07-25 o 10 35 30

**Pull request merged**